node/external/nodeApiResolver.go

```diff
@@ -197,6 +205,24 @@ func (nar *nodeApiResolver) GetInternalMiniBlock(format common.ApiOutputFormat,
 	return nar.apiInternalBlockHandler.GetInternalMiniBlock(format, decodedHash, epoch)
 }
 
+// GetGenesisNodesPubKeys will return genesis nodes public keys by shard
+func (nar *nodeApiResolver) GetGenesisNodesPubKeys() (map[uint32][][]byte, map[uint32][][]byte) {
+	eligibleNodesConfig, waitingNodesConfig := nar.genesisNodesSetupHandler.InitialNodesInfo()
```

Comment:
as mentioned in another comment, please use validator pub key converter so in the entire PR you should have `map[uint32][]string` instead of `map[uint32][][]byte`. 
you can convert the bytes in the `getInitialNodesPubKeysBytes` function

Reply:
will instantiate a pubKeyConverter in sdk also, maybe with a const for address length